cu_servicenow/cu_servicenow.module

```diff
@@ -0,0 +1,316 @@
+<?php
+
+/**
+ * @file
+ * This module sets up a boolean value for webforms to integrate with ServiceNow.
+ */
+
+require './includes/password.inc';
+
+function cu_servicenow_menu() {
+  $items = [];
+
+  $items['admin/config/content/servicenow'] = [
+    'title' => t('ServiceNow Service Account'),
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('cu_servicenow_config_form'),
+    'access arguments' => array('administer modules'),
+  ];
+
+  return $items;
+}
+
+function cu_servicenow_config_form($form, &$form_state) {
+  $form = [];
+
+  $form['sn_environment'] = [
+    '#type' => 'select',
+    '#title' => t('ServiceNow Username'),
+    '#options' => [
+      'https://coloradosandbox.service-now.com' => t('Sandbox'),
+      'https://colorado.service-now.com' => t('Production'),
+    ],
+    '#default_value' => variable_get('sn_environment', 'https://coloradosandbox.service-now.com'),
+  ];
+
+  $form['sn_disable'] = [
+    '#type' => 'select',
+    '#title' => t('Disable ServiceNow integration for this site?'),
+     '#options' => array(
+        0 => t('No'),
+        1 => t('Yes'),
+     ),
+    '#default_value' => variable_get('sn_disable', FALSE),
+  ];
+
+  return system_settings_form($form);
+}
+
+/**
+ * Implements hook_form_form_id_alter().
+ *
+ * Move the SN checkbox to the publishing tab.
+ */
+function cu_servicenow_form_webform_node_form_alter(&$form, &$form_state) {
+  $form['options']['field_cu_servicenow'] = $form['field_cu_servicenow'];
+  unset($form['field_cu_servicenow']);
+}
+
+/**
+ * Implements hook_form_alter().
+ *
+ * Add custom submit handler to send info to ServiceNow.
+ */
+function cu_servicenow_form_alter(&$form, &$form_state, $form_id) {
+  $sn_disabled = variable_get('sn_disable', TRUE);
+
+  if ($sn_disabled == FALSE) {
+    if (isset($form['#node']->webform) && $form['#node']->field_cu_servicenow['und'][0]['value'] && $form_id != 'webform_component_edit_form' && $form_id != 'webform_node_form') {
```

Comment:
You could use hook_form_FORM_ID_alter() to create two form_alter functions and simplify this conditional.